process/README.md

```diff
@@ -0,0 +1,27 @@
+![Barsoom](http://barsoom.se/barsoom.png)
+
+# Process conventions
+
+
+## Timeout
+
+We've agreed to a "timeout" convention, where you can make a "T" with your hands (or say "timeout", in an audio conference).
+
+This is intended as a shorthand to move us along – since we've all agreed on it and made it conventional, the idea is that it's instantly understood and respected, without taking offense at the brevity.
+
+In a stand-up or other meeting, it means "enough detail, let's get back on track".
+
+For a conversation taking place in our work room, it means "would you please end this conversation or take it elsewhere".
+
+
+## Stand-up
+
+We agreed on following these stand-up conventions
```

Comment:
Maybe 'we agree' instead of agreed, signaling that this is something we commit to and not just agreed on?